### DISCUSSION TITLE
Audio support?

### DISCUSSION THREAD

**Discussion by Catskill909** [skiptools/skip#131] [Q&A] [answered]
Are there plans for avfoundation  audio support? Audio sliders? etc
Looking for audio streaming and audio on demand features.

**Comment by marcprux** [comment, marked as answer]
We do have some very rudimentary AV support in the https://github.com/skiptools/skip-av project that maps some of the AVKit features to Exoplayer in Android. It is nowhere near complete, but it might serve as a start if you want to contribute improvements to the package.

You can see a simple example of using it at https://github.com/skiptools/skipapp-showcase/blob/main/Sources/Showcase/VideoPlayerPlayground.swift .

**Comment by Catskill909** [comment]
Thanks!
Great tool with lots of potential.
Will be keeping track of your progress
